op-wheel/cmd/main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/urfave/cli"
+
+	"github.com/ethereum/go-ethereum/log"
+
+	oplog "github.com/ethereum-optimism/optimism/op-service/log"
+	wheel "github.com/ethereum-optimism/optimism/op-wheel"
+)
+
+var (
+	Version   = ""
+	GitCommit = ""
```

Comment:
If we want to keep this style, we'll need to add a makefile (or can switch to buildinfo)